exporter/alertmanagerexporter/go.mod

```diff
@@ -7,15 +7,15 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.88.0
 	github.com/prometheus/common v0.45.0
 	github.com/stretchr/testify v1.8.4
-	go.opentelemetry.io/collector/component v0.88.1-0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/config/confighttp v0.88.1-0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/config/configopaque v0.88.1-0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/config/configtls v0.88.1-0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/confmap v0.88.1-0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/consumer v0.88.1-0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/exporter v0.88.1-0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/pdata v1.0.0-rcv0017.0.20231112151805-b570812b1e06
-	go.opentelemetry.io/collector/semconv v0.88.1-0.20231112151805-b570812b1e06
+	go.opentelemetry.io/collector/component v0.89.0
```

Comment:
Sorry about this, but we have a new release happening right now: 0.90.0/1.0.0. I tried to update your PR to use that, but ran into some dependency issues. Once this PR is rebased and the CI is green, I'll merge this one.

Reply:
Thank you! I have updated the exporter to use the new release. One of the checks is failing due to a post error, will retry and update here.

Reply:
@jpkrohling All checks have passed on retry, thank you.